modules/tinymce/src/core/main/ts/api/dom/StyleSheetLoader.ts

```diff
@@ -22,6 +22,8 @@ export interface StyleSheetLoader {
   unload: (url: string) => void;
   unloadAll: (urls: string[]) => void;
   _setReferrerPolicy: (referrerPolicy: ReferrerPolicy) => void;
+  appendFontsToTheGreatContainerOfOurBelovedTinyMce: (fonts: string[]) => void;
```

Comment:
I wasn't sure what'd be a good name for the document that the users had put the editor in. Help welcome.

Reply:
`hostContainer`?

Reply:
This shouldn't even be a function, in fact there's no need to touch this module at all since the font css is just another stylesheet. So it should just load it using the current functions instead.

Reply:
To clarify more, the part of code that needs to load the font CSS needs to use 2 different StyleSheetLoaders. One for the editor and one for the root document. eg:
```
editor.dom.styleSheetLoader.loadAll(fontCssUrls, ...)
editor.ui.styleSheetLoader.loadAll(fontCssUrls, ...)
```